generators/app/templates/README.md

```diff
@@ -2,16 +2,50 @@
 
 <%= projectDescription %>
 
+You can find the objective of this project and dependencies in the [ARCHITECTURE.md](ARCHITECTURE.md) file within this repository.
+
+> **Reminder:** [Making READMEs readable](https://pages.18f.gov/open-source-guide/making-readmes-readable/).
+
+## Folder Structure
+
+```
+.
+├── .github
+|   └── PULL_REQUEST_TEMPLATE.md
+├── bin
+├── config
+├── lib
+├── src
```

Comment:
Please, also comment about the `src/routes`, which contains the files that implement the controllers + API docs.
We might explain that the route path itself is generated for the programmer, using the file tree itself.

Example:
`routes/nominations/{nominationId}.js` ---Implements a controler + API Docs for route---> `/nominations/{nominationId}.js`